woof-code/rootfs-skeleton/usr/sbin/sfs_load

```diff
@@ -48,6 +48,41 @@
 #exec &>/tmp/sfsl.log ; set -x
 
 . /etc/rc.d/PUPSTATE
+
+if [ -L /bin -o -L /sbin -o -L /lib -o -L /lib64 -o -L /lib32 -o -L /libx32 ]; then
+
+ if [ "$1" ]; then
+ 
+   SFSFILE="$(basename "$1")"
+   MNTFLD="$(echo "${SFSFILE}-module" | sed -e 's#\ #_#g')"
+   mkdir -p /mnt/"$MNTFLD"
+   DEVLOOP="$(losetup -f)"
+   losetup "$DEVLOOP" "$1"
```

Comment:
Without `-r`, losetup opens the file for writing, and that causes aufs/overlay to copy the SFS to pup_rw (with PUPMODE 13), and then it gets written back to flash.

This is exactly the same bug fixed in #2986.